fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -43,10 +44,12 @@
 PROJECT_DIR = os.path.join(PKG_DIR, '..')
 ENV_DIR = os.path.join(PROJECT_DIR, 'env')
 CACHE_DIR = os.path.join(PKG_DIR, 'cache')
-RAW_RESULTS_FILENAME = 'zulip_raw_results.csv'
+# TODO: change CSV names to clarify that `zulip_raw_results.csv` is only for queried / matched messages
+RAW_RESULTS_MATCHES_FILENAME = 'zulip_raw_results.csv'
```

Comment:
TODO: change CSV names to clarify that `zulip_raw_results.csv` is only for queried / matched messages